src/main/java/networkbook/logic/commands/AddCommand.java

```diff
@@ -136,23 +136,17 @@ private Graduation addGraduation(Person personToAddInfo, EditPersonDescriptor ed
         }
         return newGraduation.orElse(oldGraduation.orElse(null));
     }
-    private Course addCourse(Person personToAddInfo, EditPersonDescriptor editPersonDescriptor)
-            throws CommandException {
-        Optional<Course> oldCourse = personToAddInfo.getCourse();
```

Comment:
Jiale raised an issue that conventionally people avoid using Optional types in Java classes. We're mostly in agreement and plan to change this after v1.2. Just letting you know!